src/server/server_family.cc

```diff
@@ -230,7 +230,7 @@ using strings::HumanReadableNumBytes;
 
 namespace {
 
-const auto kRedisVersion = "7.2.0";
+const auto kRedisVersion = "7.4.0";
```

Comment:
consider exposing it in common.h as a constant (or other header file) and then reference it in tests reduce amount of manual changes